hr_holidays_public/models/hr_employee.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2015 iDT LABS (http://www.@idtlabs.sl)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+from datetime import timedelta
+from dateutil import rrule
+from odoo import fields, models
+from odoo.tools import float_utils
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    def _get_public_holidays_leaves(self, start_dt, end_dt):
```

Comment:
should not this be `_compute_`...?

Reply:
_compute_ is used for computed fields, and from there the warning that each compute function of a field should start with "_compute"